### ISSUE TITLE
Help channel message count features

### ISSUE THREAD

**Issue by ceSchueler** [danparizher/Pax-Academia#285]
This ( #164  #167 ) should exclude messages sent in channels in the "Special Help" category (360689175944626176).
Also a command to check a users message count in help channels would be neat.

**Comment by ceSchueler** [comment]
> Absolutely not. No No No....
Best case scenario you can try to create some CRUD pages for certain fields but otherwise no
How about some _ R _ _ pages (so only Read) for all fields?

Actually, I don't know all the fields we have in the DB, is that documented somewhere, or would I need to check the code?

**Comment by sebastiaan-daniels** [comment]
> Actually, I don't know all the fields we have in the DB, is that documented somewhere, or would I need to check the code?

Yes, it is quite easy, [Download this program](https://staruml.io/download/) and open the file in `database/ERD.mdj`

> How about some _ R _ _ pages (so only Read) for all fields?

There are 5 database tables, 3 of those are directly related to staff applications. These are already part of a fully working CRUD.
The other two is the `user` table and `alerts` table.

For the `user` table, as you've suggested the only thing kept there is the message count. There will be a command to read those soon. We could do something similar for the `alerts`.

If we'd do that we would have pretty much the entire db covered.

**Comment by ceSchueler** [comment]
> Yes, it is quite easy, [Download this program](https://staruml.io/download/) and open the file in database/ERD.mdj
Ooh this looks great, thanks!

> For the user table, as you've suggested the only thing kept there is the message count. There will be a command to read those soon. We could do something similar for the alerts.
And cooldown, but I guess that would only be relevant if a user complaints that they can't submit an application and then only to see `if cooldown >0`.

> If we'd do that we would have pretty much the entire db covered.
Yeah, that would be great!

**Comment by ceSchueler** [comment]
Edited issue to reflect some additional feature requests for the staff command.
